manifests/broker/config.pp

```diff
@@ -23,12 +25,12 @@
   }
 
   file { '/opt/kafka/config/server.properties':
+    ensure  => present,
```

Comment:
This is not necessary and we do not use it for our other modules. 
https://docs.puppetlabs.com/puppet/latest/reference/type.html#file-attribute-ensure